pkg/nfd-worker/nfd-worker.go

```diff
@@ -172,6 +177,35 @@ func newDefaultConfig() *NFDConfig {
 	}
 }
 
+func (i *infiniteTicker) Reset(d time.Duration) {
+	if d > 0 {
+		i.Ticker.Reset(d)
+	} else {
+		// If the sleep interval is not a positive number the ticker will act as if it was set to an infinite duration by not ticking.
+		i.Ticker.Stop()
```

Comment:
Could you format this comment to fit in 80 columns, see how other comments when too long they are several lines. makes it easier to read

Reply:
I changed the if-else to a switch and made the comment fit into 80 columns